jobs/build.Jenkinsfile

```diff
@@ -23,8 +23,8 @@ properties([
              defaultValue: '',
              trim: true),
       string(name: 'ADDITIONAL_ARCHES',
-             description: 'Space-separated list of additional target architectures',
-             defaultValue: pipecfg.additional_arches.join(" "),
+             description: 'Override default additional target architectures (space-separated)',
+             defaultValue: "",
```

Comment:
we've actually been using this (overriding the default) today. Right now the pipeline is configured to build for `ppc64le` too but we tell ourselves when we go to make official builds to remove the `ppc64le` from the list.

Now there will be no list to remove an item from, but rather an empty field. It's not a big deal but I kind of prefer the other UX.